src/style.css

```
@@ -0,0 +1,24 @@
+body {
+  margin: 0;
+}
+  
+.stars {
+  display: flex;
+}
+  
+.stars-block {
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 15px;
+  height: 15px;
+  margin-right: 4px;
+}
+  
+.stars-first div:nth-child(-n+1),
```

```suggestion
.stars-first .stars-block:nth-child(-n+1),
```
Try to avoid stylizing tags. Always use classes for this. Tags stylizing is OK when you reset default browser styles, like margin: 0 for BODY tag, which has 8px margins. Setting width and height for all DIV tags is bad practice